src/index.html

```diff
@@ -16,7 +37,338 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="top-bar">
+          <a
+            href="#"
+            class="top-bar__logo"
+          >
+            <img
+              src="./images/header/logo.png"
+              alt="Bang & Olufsen logo"
+              class="top-bar__logo-img"
+            />
+          </a>
+          <div class="top-bar__icons">
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--phone"
+            ></a>
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            ></a>
+          </div>
+        </div>
+        <div class="header__bottom">
+          <h1 class="header__title">
+            Your senses.
+            <br />
+            Set free.
+          </h1>
+        </div>
+      </div>
+    </header>
+    <aside
```

Comment:
It’s better to leave a empty line between sections for easier reading of the code